src/core/components/learners/ProfileCourses.vue

```diff
@@ -109,11 +114,22 @@ export default {
     }
   },
   methods: {
-    goToBlendedCourseProfileAdmin (row) {
-      if (!this.isVendorInterface) return;
+    goToBlendedCourseProfile (props) {
+      if (!this.isVendorInterface) {
+        props.expand = !props.expand;
+        return;
+      }
+
+      if (props.row.subProgram.isStrictlyELearning) {
+        this.$router.push({
```

Comment:
Je crois qu'il faut que tu ajoutes return devant sinon on arrive encore sur la formation blended

Reply:
done

Reply:
Tu peux mettre le return directement devant le `this.$router.push`;